templates/telco/index.html

```diff
@@ -0,0 +1,242 @@
+                </div>
+                <div class="col-6 p-card">
+                    <h3>Cloud-side latency optimisation</h3>
+                    <hr>
+                    <p>Anbox Cloud leverages hardware acceleration for rich graphical rendering, animations and effects. The platform also enables high-performance video encoding and decoding and streaming. To minimise latency bottlenecks, compute intensive tasks like encoding and decoding are pooled on GPUs on the server side to offload mobile devices.</p>
+                </div>
+            </div>
+        </section>
+
+        <section>
+            <div class="p-strip--light u-no-padding--bottom">
+                <div class="row">
+                    <div class="col-6">
+                        <h2>What’s under the hood</h2>
+                        <h4>Anbox Cloud builds on a powerful stack of open-source software to deliver Android app streaming from the cloud.</h4>
+                    </div>
+                </div>
+            </div>
+
+            <div class="p-strip--light">
+                <div class="row u-vertically-center">
+                    <div class="col-7">
+                        <h3>Android in containers</h3>
+                        <p>Unlike Android emulators that make use of virtual machines, Anbox Cloud leverages system containers through LXD, the next generation system container manager. It offers a user experience similar to virtual machines but using lightweight Linux containers instead manageable via a REST API. It also exposes GPUs through a passthrough. You will therefore take advantage of any hardware accelerators to deliver rich mobile experiences.</p>
+                    </div>
+                    <div class="col-5 u-align--center u-hide--small">
+                        {{ image (
+                            url="https://assets.ubuntu.com/v1/ac9a0e00-lxd_primary.svg",
+                            alt="",
+                            width="229",
+                            height="75",
+                            hi_def=True,
+                            loading="auto"
+                            ) | safe
+                        }}
+                    </div>
+                </div>
+                <div class="u-fixed-width">
+                    <hr class="p-separator">
+                </div>
+                <div class="row u-vertically-center">
+                    <div class="col-7">
+                        <h3>Software orchestration</h3>
+                        <p>Streaming Android apps to thousands of mobile and desktop clients simultaneously requires orchestration and monitoring of several software components. This involves streaming servers, image repositories, applications, a dashboard application for management and more. Thanks to Juju and MAAS, you can deploy your stack automatically anywhere. </p>
```

Comment:
`more`, `Juju` and `MAAS` should be links

Reply:
What should 'more' link to?

Reply:
https://anbox-cloud.io/docs/overview